ext/cl_khr_int32_atomics.txt

```diff
@@ -0,0 +1,228 @@
+// Copyright 2017-2018 The Khronos Group. This work is licensed under a
+// Creative Commons Attribution 4.0 International License; see
+// http://creativecommons.org/licenses/by/4.0/
+
+[[cl_khr_int32_atomics]]
+== 32-bit Atomics
+
+This section describes the extensions *cl_khr_global_int32_base_atomics*, *cl_khr_global_int32_extended_atomics*, *cl_khr_local_int32_base_atomics*, and *cl_khr_local_int32_extended_atomics*.
+These extensions allow atomic operations to be performed on 32-bit signed and unsigned integers in global and local memory.
+
+These extensions became core features in OpenCL 1.1.
```

Comment:
Or, if we want to address the below TODO as well:

```suggestion
These extensions became core features in OpenCL 1.1, except the built-in atomic function names are changed to use the **atomic_** prefix instead of **atom_** and the volatile keyword was added to the pointer parameters _p_.
```